python/nav/buildconf.py

```diff
@@ -9,4 +9,10 @@
 webrootdir = os.path.join(datadir, "www")
 djangotmpldir = os.path.join(datadir, "templates")
 docdir = os.path.join(datadir, "doc")
-VERSION = pkg_resources.get_distribution("nav").version
+
+try:
+    VERSION = pkg_resources.get_distribution("nav").version
+except pkg_resources.DistributionNotFound:
+    # If we're not installed, try to get the current version from Git tags
```

Comment:
If we are not installed, but in a tree made by unpacking an sdist, then what? No git tags available.

Reply:
Yep, then we are screwed. We always were. Unless setuptools_scm has some fallback to a cached version number somewhere.. can't recall atm whether that can be done...

Reply:
This is one reason I don't like setuptools_scm :)